tools/op.sh

```diff
@@ -21,14 +23,25 @@ function op_first_install() {
 # be default, assume openpilot dir is in current directory
 OPENPILOT_ROOT=$(pwd)
 function op_check_openpilot_dir() {
-  (set -e
+  while [[ "$OPENPILOT_ROOT" != '/' ]];
```

Comment:
shouldn't need to search since you know the script is in openpilot

https://github.com/commaai/openpilot/blob/31036771ee8ba71e3fbe7c7f08aad3b5be0127e6/launch_chffrplus.sh#L9

Reply:
this allow to run `op cmd` in `/tmp/openpilot/selfdrive/ui/...` even if you use the `op` tool from `$HOME/openpilot`